cdap-common/src/test/java/co/cask/cdap/common/test/AppJarHelper.java

```diff
@@ -102,7 +102,7 @@ public static Location createDeploymentJar(LocationFactory locationFactory, Clas
           }
 
           // TODO: this is due to manifest possibly already existing in the jar, but we also
-          // create a manifest programatically so it's possible to have a duplicate entry here
+          // create a manifest programmatically so it's possible to have a duplicate entry here
```

Comment:
I just did a find/replace on the whole code base. That is how I found that we had an existing utility method. Thought it's best to just fix all misspellings.